pkg/cmd/device_groups/list/list.go

```diff
@@ -0,0 +1,108 @@
+package list
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+
+	"github.com/MakeNowJust/heredoc"
+	table "github.com/MaxwelMazur/tablecli"
+	msg "github.com/aziontech/azion-cli/messages/device_groups"
+	api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/contracts"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	var edgeApplicationID int64 = 0
+	opts := &contracts.ListOptions{}
+	cmd := &cobra.Command{
+		Use:           msg.DeviceGroupsListUsage,
+		Short:         msg.DeviceGroupsListShortDescription,
+		Long:          msg.DeviceGroupsListLongDescription,
+		SilenceUsage:  true,
+		SilenceErrors: true, Example: heredoc.Doc(`
+        $ azioncli device-groups list -a 16736354321
```

Comment:
Can you check the formatting here? it might be a problem when we use the --help flag in terminal